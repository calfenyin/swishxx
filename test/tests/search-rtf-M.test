search | | -i rtf.index -M | | 0
