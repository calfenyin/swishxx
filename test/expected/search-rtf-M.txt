author
category
comment
company
keywords
manager
operator
subject
title
